samples/scripts/syncAnnotationsScript.py

```diff
@@ -0,0 +1,261 @@
+            )
+            name_map = {}
+            for item in folder_list:
+                name_map[item["name"]] = item["_id"]
+            if name in name_map.keys():
+                upload_folder = name_map[name]
+            else:
+                new_folder = gc.sendRestRequest(
+                    "POST",
+                    f'dive_dataset?cloneId={annotation["girderId"]}&parentFolderId={parent_folder}&name={name}',
+                )
+                upload_folder = str(new_folder["_id"])
+
+        # Now we upload the file and trigger a post process to view the data
+        gc.uploadFileToFolder(upload_folder, upload_file)
+        gc.sendRestRequest(
+            "POST",
+            f"dive_rpc/postprocess/{upload_folder}",
+            data={"skipTranscoding": True, "skipJobs": True},
+        )
+
+
+# uploads annotations to a user public folder
+def clone_and_upload_annotations(
+    gc: girder_client.GirderClient, annotations, rootFolder
+):
+    for annotation in annotations:
+        upload_folder = annotation["girderId"]
+        upload_file = annotation["file"]
+        name = f"{os.path.basename(annotation['file']).replace('.csv','')}"
+        if annotation["label"] != default_label:  # now we change the name
+            name = f"{name} - {annotation['label']}"
+        new_folder = gc.sendRestRequest(
+            "POST",
+            f'dive_dataset?cloneId={annotation["girderId"]}&parentFolderId={rootFolder}&name={name}',
+        )
+        upload_folder = str(new_folder["_id"])
+        gc.uploadFileToFolder(upload_folder, upload_file)
+        gc.sendRestRequest(
+            "POST",
+            f"dive_rpc/postprocess/{upload_folder}",
+            data={"skipTranscoding": True, "skipJobs": True},
+        )
+
+
+# returns the public folder of the user who logged in
+def get_public_folder(gc: girder_client.GirderClient):
+    current_user = gc.sendRestRequest("GET", "user/me")
+    userId = current_user["_id"]
+    folders = gc.sendRestRequest(
+        "GET",
+        f"folder?parentType=user&parentId={userId}&text=Public&limit=50&sort=lowerName&sortdir=1",
+    )
+    if len(folders) > 0:
+        uploadFolder = folders[0]["_id"]
+    else:
+        print("No folder found for the user")
+    return uploadFolder
+
+
+def ask_question(prompt):
+    while True:
+        response = input(prompt).strip().lower()
+        return response
+
+
+def ask_yes_no_question(prompt):
+    while True:
+        response = input(prompt).strip().lower()
+        if response == "y":
+            return True
+        elif response == "n":
+            return False
+        else:
+            print("Invalid input. Please enter 'y' or 'n'.")
+
+
+@click.command(name="LoadData", help="Load in ")
+@click.argument(
+    "folder"
+)  # a local folder to search for mp4 video files and json/csv files.
+@click.argument("girder_id", required=False)
```

Comment:
Preferably more of  description here.


```suggestion
@click.command(name="LoadData", help="Load in annotation from local directory ")
@click.argument(
    "folder",
    help="Path of directory containing the annotation csv"
)  # a local folder to search for mp4 video files and json/csv files.
@click.argument("girder_id", required=False, help="girder_id of the destination directory on DIVE")
```